src/extension/ui.js

```diff
@@ -22,12 +24,51 @@ let updateInProgress = false;
 
 /**
  * @typedef {Object} Context
- * @prop {number} id The tab ID
- * @prop {string} url The URL
+ * @prop {string} [url] The URL
  * @prop {Object} [config] The project config
+ * @prop {Object[]} [matches] The config matches
```

Comment:
Should this be [OptionsConfig](https://github.com/adobe/aem-sidekick/blob/main/src/extension/types/typedefs.js#L35) ?

Reply:
Possibly 😉